cmd/cerbosctl/inspect/internal/print.go

```diff
@@ -86,23 +86,43 @@ func printJSON(w io.Writer, results []*responsev1.InspectPoliciesResponse_Result
 }
 
 func printTable(w io.Writer, noHeaders bool, results []*responsev1.InspectPoliciesResponse_Result) {
-	tw := printer.NewTableWriter(w)
-	if !noHeaders {
-		tw.SetHeader([]string{"POLICY ID", "ACTIONS", "VARIABLES"})
-	}
-
 	for _, result := range results {
+		attributes := make([]string, len(result.Attributes))
+		for idx, attribute := range result.Attributes {
+			switch attribute.Kind {
+			case responsev1.InspectPoliciesResponse_Attribute_KIND_PRINCIPAL_ATTRIBUTE:
+				attributes[idx] = fmt.Sprintf("%s.attr.%s", conditions.CELPrincipalAbbrev, attribute.Name)
+			case responsev1.InspectPoliciesResponse_Attribute_KIND_RESOURCE_ATTRIBUTE:
+				attributes[idx] = fmt.Sprintf("%s.attr.%s", conditions.CELResourceAbbrev, attribute.Name)
+			default:
+				attributes[idx] = attribute.Name
+			}
+		}
+
 		variables := make([]string, len(result.Variables))
 		for idx, variable := range result.Variables {
 			variables[idx] = variable.Name
 		}
 
-		tw.Append([]string{
-			result.PolicyId,
-			strings.Join(result.Actions, separator),
-			strings.Join(variables, separator),
-		})
-	}
+		var row string
+		if noHeaders {
+			row = fmt.Sprintf(
+				"%s\n%s\n%s\n%s",
+				result.PolicyId,
+				strings.Join(result.Actions, separator),
+				strings.Join(attributes, separator),
+				strings.Join(variables, separator),
+			)
+		} else {
+			row = fmt.Sprintf(
+				"%s\n%s\n%s\n%s",
+				fmt.Sprintf("POLICY ID : %s", result.PolicyId),
+				fmt.Sprintf("ACTIONS   : %s", strings.Join(result.Actions, separator)),
+				fmt.Sprintf("ATTRIBUTES: %s", strings.Join(attributes, separator)),
+				fmt.Sprintf("VARIABLES : %s", strings.Join(variables, separator)),
```

Comment:
```suggestion
				"POLICY ID : " + result.PolicyId,
				"ACTIONS   : " + strings.Join(result.Actions, separator),
				"ATTRIBUTES: " + strings.Join(attributes, separator),
				"VARIABLES : " + strings.Join(variables, separator),
```

Reply:
Some policies don't have attributes, variables or actions. It'd look nicer if we don't print headers for values that don't exist.